vocabulary/educational_institution.yml

```diff
@@ -0,0 +1,23 @@
+id: "educational_institution"
+title: "Educational Institution"
+description: ""
+type: "vocabulary"
+properties:
+  name:
+    type: "string"
+    title: "pavadinimas"
+  id:
+    type: "integer"
+    title: "kodas"
+  address:
+    type: "string"
+    title: "adresas"
+  management_email:
+    type: "string"
+    title: "vad_email"
```

Comment:
Titles should be human readable, meaning, no underscores and should start from capital letter.